Frontend/CO.CDP.OrganisationApp/Pages/Shared/_FlashMessage.cshtml

```diff
@@ -7,28 +7,30 @@
 {
     var message = flashMessageService.GetFlashMessage(FlashMessageType.Success);
     if(message != null) {
-        <div class="govuk-notification-banner govuk-notification-banner--success" role="alert" aria-labelledby="govuk-notification-banner-title" data-module="govuk-notification-banner">
-            <div class="govuk-notification-banner__header">
-                <h2 class="govuk-notification-banner__title" id="govuk-notification-banner-title">
-                    @if (message.Title != null)
-                    {
-                        @message.Title
-                    }
-                    else
+        <div class="govuk-grid-column-two-thirds govuk-grid-row">
```

Comment:
Columns must sit inside a `govuk-grid-row` div. See [understanding-the-grid-system](https://design-system.service.gov.uk/styles/layout/#using-the-grid-system)

```
<div class="govuk-grid-row">
  <div class="govuk-grid-column-two-thirds">
    ...
  </div>
</div>
```

Reply:
done